specification/cognitiveservices/data-plane/Language/preview/2022-04-01-preview/examples/SuccessfulAnalyzeConversationJobsCancelRequest.json

```diff
@@ -0,0 +1,15 @@
+{
+  "parameters": {
+    "Ocp-Apim-Subscription-Key": "{API key}",
+    "ApiVersion": "2022-03-01-preview",
+    "Endpoint": "{Endpoint}",
+    "jobId": "c0f2a446-05d9-48fc-ba8f-3ef4af8d0b18"
+  },
+  "responses": {
+    "202": {
+      "headers": {
+        "Operation-Location": "{Endpoint}/language/analyze-conversation/jobs/{jobId}?api-version=2022-04-01-preview"
```

Comment:
I just noticed that we are using `analyze-conversation` in the path for these API operations, but previous conversational APIs use `analyze-conversations` (with an "s").  

I think we may want to align to the use of `analyze-conversations` instead for consistency.  Let me know if there was a reason for it to be without the "s".

Reply:
Updated to May version.